nautobot/dcim/forms.py

```diff
@@ -1902,6 +1902,7 @@ class DeviceForm(LocatableModelFormMixin, NautobotModelForm, TenancyForm, LocalC
             attrs={
                 "disabled-indicator": "device",
```

Comment:
The `disabled-indicator` attr is *supposed* to make it so that any RU that have an associated value for their `device` show as disabled in the menu; however, looking at the API response, there is no `device` field in the data:

![image](https://github.com/user-attachments/assets/a73a1f1d-d842-48ac-9e1a-b239b1c7efe6)

You might try as an alternate approach setting `"disabled-indicator": "occupied"` and see if that makes it so that the occupied RU still appear in the menu but as disabled entries?